docs/02-app/01-building-your-application/03-rendering/index.mdx

```diff
@@ -71,7 +71,7 @@ When working in these environments, it's helpful to think of the flow of the cod
 
 {/* Diagram: Response flow */}
 
-If you need to access the server from the client, you send a **new** request to the server rather than re-use the same request. This makes it easier understand where to render your components and where you to place the Network Boundary.
+If you need to access the server from the client, you send a **new** request to the server rather than re-use the same request. This makes it easier understand where to render your components and where to place the Network Boundary.
```

Comment:
```suggestion
If you need to access the server from the client, you send a **new** request to the server rather than re-use the same request. This makes it easier to understand where to render your components and where to place the Network Boundary.
```